samples/cpp/calibration.cpp

```diff
@@ -154,8 +158,7 @@ static bool runCalibration( vector<vector<Point2f> > imagePoints,
     objectPoints.resize(imagePoints.size(),objectPoints[0]);
 
     double rms = calibrateCamera(objectPoints, imagePoints, imageSize, cameraMatrix,
-                    distCoeffs, rvecs, tvecs, flags|CALIB_FIX_K4|CALIB_FIX_K5);
-                    ///*|CALIB_FIX_K3*/|CALIB_FIX_K4|CALIB_FIX_K5);
+                    distCoeffs, rvecs, tvecs, flags | CALIB_USE_LU);
```

Comment:
`CALIB_USE_LU` is used in OpenCV 4.x: https://github.com/opencv/opencv/blob/1fd45a1b8580ee90fa63c6106c78217c846247e4/samples/cpp/calibration.cpp#L173

Normally no need to pass `CALIB_FIX_K4|CALIB_FIX_K5` (`CALIB_RATIONAL_MODEL` should be explicitly passed), moreover `CALIB_FIX_K6` should be missing.